docs/testing-operators.md

```diff
@@ -1,120 +1,404 @@
+For simplicity, the following commands will clone all of the repositories above:
 
-The scorecard assigns points to each passing component of the scorecard. The total number of points is a function of several factors, ex. number of CRD's, but a weighted total percentage is calculated for the overall test run. It is possible to get a score of 100%, but operators are not expected to achieve this.
+```
+git clone https://github.com/operator-framework/operator-marketplace.git
+git clone https://github.com/operator-framework/operator-courier.git
+git clone https://github.com/operator-framework/operator-lifecycle-manager.git
+```
+
+Before you begin your current working dir should look like the following, with `my-operator` as an example for the name of your bundle:
 
-`test-operator` injects a scorecard proxy container and volume into an operators' CSV manifest before deployment; this is necessary to get API server logs, from which the scorecard determines runtime test results. These modifications are not persistent, as they're only needed for testing.
+```
+my-operator
+operator-marketplace
+operator-courier
+operator-lifecycle-manager
+```
 
-**Note**: no explicit number of points or percentage is necessary to achieve before merging _yet_. These are suggestions to improve your operator.
+### Tools
 
 #### operator-courier
 
-The [`operator-courier verify`][courier] command verifies that a set of files is valid and can be bundled and pushed to [quay.io][quay]. Read the [docs][courier-docs] for more information.
+`operator-courier` is used for metadata syntax checking and validation. This can be installed directly from `pip`:
+
+```
+pip3 install operator-courier
+```
+
+#### Quay Login
+
+In order to test the Operator installation flow, store your Operator bundle on [quay.io](https://quay.io). You can easily create an account and use the free tier (public repositories only). To upload your Operator to quay.io a token is needed. This only needs to be done once and can be saved locally. The `operator-courier` repository has a script to retrieve the token:
+
+```
+./operator-courier/scripts/get-quay-token
+
+Username: johndoe
+Password: 
+{"token": "basic abcdefghijkl=="}
+```
+
+A token takes the following form and should be saved in an environment variable:
+
+```
+export QUAY_TOKEN="basic abcdefghijkl=="
+```
+### Linting
+
+`operator-courier` will verify the fields included in the Operator metadata (CSV). The fields can also be manually reviewed according to [the operator CSV documentation](https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md).
```

Comment:
I understand why this is in this section, but I still want to argue against *advertising* the courier as a linter.

Reply:
If we have something better, sure. Until then, it's the only thing we got that does some validation off-cluster.

Reply:
I would still prefer we stopped using the term `linter`. When we advertise it as a linter, it is expected to function as a linter, and we are forced to support it as a linter.